### DISCUSSION TITLE
Resources - Python Coding and Package Development

### DISCUSSION THREAD

**Discussion by OlivierBinette** [OlivierBinette/VisTree#15] [General]
This discussion thread is meant to share any resource related to coding, package development, git, IDEs, etc.

**Comment by oueini** [comment]
For anyone who is starting out, this tutorial helped me understand better how to create a graph from scratch:

https://igraph.org/python/tutorial/0.9.8/tutorial.html

Hope it helps!

**Reply by szhorvat** [reply]
The latest version is at https://python.igraph.org/en/stable/tutorial.html
